grpc-google-cloud-datastore-v1/src/main/java/com/google/datastore/v1/DatastoreGrpc.java

```diff
@@ -0,0 +1,1053 @@
+/*
```

Comment:
I am surprised to see this get generated for only the admin client - is this correct?

Reply:
Owl seemed to be trying to add the repo as a dependency, but I don't see any reason that is actually true. I may have just misconfigured the owl configuration during that run. Removed it, we'll see if owl tries to put it back.